packages/devtools_app/lib/src/flutter/scaffold.dart

```diff
@@ -175,11 +185,12 @@ class DevToolsScaffoldState extends State<DevToolsScaffold>
         Align(
           alignment: Alignment.topLeft,
           child: Padding(
-            padding: const EdgeInsets.all(32.0),
+            padding: DevToolsScaffold.borderInsets,
```

Comment:
We have a hack in the flame chart because  PointerHoverEvent.localPosition doesn't actually return local position. Can you change the value of `flameChartContainerOffset` in `flame_chart.dart` to 17.0? The value was originally hard coded to 33.0 and I'm assuming changing this value from 32.0 to 16.0 will affect the margin for all pages.

Reply:
Good catch - done.